prismic-model/lintPrismicData.ts

```diff
@@ -277,6 +302,22 @@ async function run() {
         console.log(`- ${msg}`);
       }
       console.log('');
+
+      // Send an alert to Editors if anything is found on a GitHub Action run
+      // https://github.com/wellcomecollection/wellcomecollection.org/actions/workflows/prismic-linting.yml
+      if (isGitHubAction && slackWebhookUrl) {
```

Comment:
As with the other prismic linting PR, could we rely solely on the `slackWebhookUrl` to signal we should send to slack?

Reply:
The webhook url is always the same, and we don't want this to run when we run the report locally for our own testing, so I don't think it applies?

Reply:
Hadn't noticed that we're pulling from secrets manager here for `SLACK_WEBHOOK_GA_PRISMIC_LINTING_URL` , I wonder if either that could be externalised to another [GHA step](https://github.com/aws-actions/aws-secretsmanager-get-secrets) or move the secret retrieval inside the check for `isGitHubAction` as it is unnecessary otherwise?

Is it useful to pass the webhook URL through an env var here too?

Reply:
OK discussed this with @agnesgaroux. We believe that because we already use a GitHub secret in the `prismic-linting.yml` workflow/action, it makes sense to just move the secret to GitHub so both referenced secrets live in the same space. I'll just confirm it doesn't print it when it passes it to the script, but otherwise it's much simpler.

Reply:
Confirmed non printed
```Run docker compose run prismic_model yarn lintPrismicData slackWebhookUrl=***```

I've scheduled the AWS secret for deletion in 7 days.